weave/trace/weave_client.py

```diff
@@ -749,6 +749,7 @@ def finish_call(
             if isinstance(usage, dict) and isinstance(model, str):
                 summary["usage"] = {}
                 summary["usage"][model] = {"requests": 1, **usage}
+        call.summary = summary
```

Comment:
Moving summary assignment from below to keep that logic together.

Reply:
makes sense